### PR TITLE
Community Spotlight: Reducing search response time by 99.98% with Xata

### PR DESCRIPTION
**Pull request by alexfrancoeur**
## Summary

Putting the spotlight on Parker Dinkins, a young founder with a love for geospatial data who recently launched OpenSourceGISData.com.

External draft: https://www.notion.so/xata/External-Reducing-search-response-time-by-99-98-with-Xata-c65f0fca3ad3465b944fd7def7b76ec5

Illustration request: https://github.com/xataio/mdx-blog/issues/171

### Blog Release Checklist

- \[ ] Release on: 01/09/24
- \[ ] Published on dev.to (connected to Xata org)
- \[ ] OG image included
- \[ ] Social media is scheduled for this post

### CODE REVIEW

**Comment by alexfrancoeur** [comment]
Response from Parker:

> Hello Alex,
> I did have a great Thanksgiving, I hope you did too. Don't worry about it, I know you probably have a lot on your plate right now. The first 
> draft looks awesome! Thank you for choosing me and writing this. There is a few minor things I would like to change. I can make the edits in the Notion if that's alright with you. 
> 
> Answers to your questions:
> 1. The image of me with the globe behind is perfect.
> 2. The screenshots looks good. I am planning on remaking my landing page next week since my semester is over, so it would be cool to update that and the demo video with the new one. 
> 3. Currently there is 740k layers from 1.7k sources.
> 4. The GIS Twitter account is: @GISDataIO ; LinkedIn is: https://www.linkedin.com/company/gisdataio/ ; GitHub: https://github.com/OmniverseXYZ
> 5. I am starting my Senior year at UCF studying Environmental Studies, minoring in Urban and Regional Planning and GIS. Seems like you covered all of this in the article and it looks great. 
> 6. Omniverse.xyz is the company/organization that I started. It is the overarching brand that owns the GIS platform and other products I plan develop in the future. It is worth mentioning in the paragraph about the bigger vision as the Omniverse.xyz GIS Platform (GISData.io, GISAPI.io, and GISMap.io). 
> 7. I recently got accepted in the UCF UpStarts program. It is a Venture Incubator Program for student entrepreneurs. https://cel.ucf.edu/upstarts-incubator-at-ucf/
> 
> That schedule sounds perfect to me! Right after the new year seems like the best time to publish it. Again I really appreciate you showing interesting and doing this for me. 
> 
> I made a few minor edits in the notion. I changed to subtitle link to GISData.io and changed the link on my name to my LinkedIn Profile. I also changed the OSGD Platform to Omniverse.xyz GIS Platform and added a link to the GISAPI.io docs. 
> 
> I will make the new landing page this week and follow up with you.
> 
> Thank you,
> Parker 

Noting that I'll need to update the blog to accommodate his changes to the external notion and new screenshots with an updated landing page.

File: community-spotlight-reducing-search-response-time-by-99-percent.mdx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  alt: Community Spotlight
+author: Alex Francoeur
+date: 01-09-2024
+tags: ['community']
+published: true
+slug: community-spotlight-reducing-search-time
+ogImage: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/community-spotlight-reducing-search-response-time-by-99-percent.jpg
+---
+
+[Parker Dinkins](https://www.linkedin.com/in/john-parker-dinkins/), is a young entrepreneur out of Florida. As a self taught developer, he initially founded a startup with some friends back in 2022 called Precision Ecology. It was a digital mapping data solutions company focused on providing data for sustainable development and environmental management. They were flying a [Wingtra](https://wingtra.com/) drone collecting aerial images across large amounts of acreage and Parker was responsible for the processing of the data. This is where he fell in love with geospatial data and GIS.
```
Comment (joan-ing) [inline community-spotlight-reducing-search-response-time-by-99-percent.mdx:15]:
```suggestion
[Parker Dinkins](https://www.linkedin.com/in/john-parker-dinkins/), a Florida-based entrepreneur and self-taught developer, launched his startup, Precision Ecology, in 2022 with a group of friends. The company focused on digital mapping and data solutions, targeting sustainable development and environmental management. They used a Wingtra drone for gathering aerial images over large tracts of land. Parker's responsibility in processing these images led to his deep interest in geospatial data and GIS.
```

File: community-spotlight-reducing-search-response-time-by-99-percent.mdx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+After Precision Ecology didn’t pan out, he decided to go back to to school and study Environmental Studies to focus on GIS. So what does he do when he’s a student back at school? Launch another startup of course.
```
Comment (joan-ing) [inline community-spotlight-reducing-search-response-time-by-99-percent.mdx:17]:
```suggestion
When Precision Ecology didn't pan out, Parker switched gears and went back to school to dig into Environmental Studies and focus his studies on GIS.

So what does an experienced entrepreneur like Parker do when he’s back at school? Launch another startup, of course.
```

File: community-spotlight-reducing-search-response-time-by-99-percent.mdx [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+Cue [OpenSourceGISData.com](http://OpenSourceGISData.com) (OSGD), Parker's latest venture. OSGD is a website that contains geoportals and ArcREST servers for all the major federal agencies, states, counties, and cities. After seeing some popularity on Reddit and healthy daily traffic, he feels he scratched an itch for geospatial professionals.
+
+![Open Source GIS Data](/images/cs-osgd.png)
+
+About a year after launching the OSGD website, he officially launched a sister search engine [GISData](https://gisdata.io/). GISData has indexed over 740,000 layers from 1,700 different sources and counting. The goal was to provide a single source of truth where users can search across all disparate data sources. And so far, it’s proven to be a hit.
+
+## An evolving architecture for geospatial data search
+
+Balancing straight A’s at the University of Central Florida with a budding startup is no easy task, but some how Parker is managing it just fine.
```
Comment (joan-ing) [inline community-spotlight-reducing-search-response-time-by-99-percent.mdx:31]:
```suggestion
Balancing straight A’s at the University of Central Florida with a budding startup is no easy task, but somehow Parker is managing it just fine.
```

File: community-spotlight-reducing-search-response-time-by-99-percent.mdx [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+![Geospatial Search](/images/cs-gis-search.png)
+
+The search behind GISData is built on a modern technology stack. It’s currently using [Dagster](https://dagster.io/) for orchestration, Python for indexing scripts, Xata as the database, JavaScript and [UIkit](https://getuikit.com/) for the frontend and Vercel for application deployment. This stack is optimized for speed and minimal operational maintenance.
+
+## Just getting started
+
+Parker has a larger vision for this platform beyond the GISData search engine. He’s planning to roll out a free and paid version of the [GIS API](https://docs.gisapi.io/) that enables developers to integrate geospatial data discovery into their own applications. For less technical users, he’s aiming to a create a simple mapping platform where users with little to no experience can find data and create simple maps for their own personal use case. We also discussed the possibility of introducing an AI geo assistant, which is another avenue of monetization. While he’ll likely be focusing on the next few months on foundational elements to support those features, there’s a bright future ahead for the [Omniverse.xyz](https://Omniverse.XYZ) GIS platform.
```
Comment (joan-ing) [inline community-spotlight-reducing-search-response-time-by-99-percent.mdx:45]:
```suggestion
Parker has a larger vision for this platform beyond the GISData search engine. He’s planning to roll out a free and paid version of the [GIS API](https://docs.gisapi.io/) that enables developers to integrate geospatial data discovery into their own applications. For less technical users, he’s aiming to create a simple mapping platform where users with little to no experience can find data and create simple maps for their own personal use case. We also discussed the possibility of introducing an AI geo assistant, which is another avenue of monetization. While he’ll likely be focusing on the next few months on foundational elements to support those features, there’s a bright future ahead for the [Omniverse.xyz](https://Omniverse.XYZ) GIS platform.
```

**Review by joan-ing — APPROVED** [review]
looks great @Alex! Just a couple minor suggestions.

File: community-spotlight-reducing-search-response-time-by-99-percent.mdx [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+  alt: Community Spotlight
+author: Alex Francoeur
+date: 01-09-2024
+tags: ['community']
+published: true
+slug: community-spotlight-reducing-search-time
+ogImage: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/community-spotlight-reducing-search-response-time-by-99-percent.jpg
+---
+
+[Parker Dinkins](https://www.linkedin.com/in/john-parker-dinkins/), a Florida-based entrepreneur and self-taught developer, launched his startup, Precision Ecology, in 2022 with a group of friends. The company focused on digital mapping and data solutions, targeting sustainable development and environmental management. They used a Wingtra drone for gathering aerial images over large tracts of land. Parker's responsibility in processing these images led to his deep interest in geospatial data and GIS.
```
Comment (alexfrancoeur) [inline community-spotlight-reducing-search-response-time-by-99-percent.mdx:15]:
```suggestion
[Parker Dinkins](https://www.linkedin.com/in/john-parker-dinkins/), a Florida-based entrepreneur and self-taught developer, launched his startup, Precision Ecology, in 2022 with a group of friends. The company focused on digital mapping and data solutions, targeting sustainable development and environmental management. They used a [Wingtra](https://wingtra.com/) drone for gathering aerial images over large tracts of land. Parker's responsibility in processing these images led to his deep interest in geospatial data and GIS.
```

**Pull request merged**